docs/concepts/migration/index.mdx

```diff
@@ -7,7 +7,7 @@ description: "Learn more about the reasons to move to Sentry's SaaS solution, wh
 Sentry offers a cloud-hosted, software-as-a-service (SaaS) solution in addition to a self-hosted solution, which are both functionally the same. However, many customers find that self-hosted Sentry can quickly become expensive to maintain, scale, and support, making our SaaS product the better and less costly option. To facilitate moving from self-hosted to SaaS, we provide a self-serve process known as "relocation".
 
 <Alert>
-Check out this video on [**Migrating to Sentry SaaS**](https://sentry.io/resources/migrate-to-sentry-saas-workshop/) to learn about our relocation tooling.
+Check out this video on (<a href="https://sentry.io/resources/migrate-to-sentry-saas-workshop/" className="plausible-event-name=migrating+to+saas+video">**Migrating to Sentry SaaS**</a>) to learn about our relocation tooling.
```

Comment:
```suggestion
Check out this video on <a href="https://sentry.io/resources/migrate-to-sentry-saas-workshop/" className="plausible-event-name=migrating+to+saas+video">**Migrating to Sentry SaaS**</a> to learn about our relocation tooling.
```

I'm seeing the parentheses around the link on https://sentry-docs-git-add-plasuible-tracking-link-to-video.sentry.dev/concepts/migration/